packages/zero-runtime/src/utils/generateCss.ts

```diff
@@ -21,3 +22,16 @@ export function generateThemeTokens(theme: BaseTheme) {
   const { styles } = serializeStyles([stylesheetObj]);
   return styles;
 }
+
+export function generateThemeTokens(theme: BaseTheme) {
+  if (!theme || typeof theme !== 'object') {
+    return {};
+  }
+  // is created using extendTheme
+  if ('vars' in theme && theme.vars) {
+    return {
+      vars: theme.vars,
```

Comment:
I think we should expose serializable properties of the theme. There are cases where you need to use raw value, e.g. shadow. This is an [example](https://mui.com/joy-ui/customization/theme-shadow/#customizing-shadows-on-an-element) from Joy UI but it's not limited to.

Reply:
Is this something that can be assigned to a css variable? If yes, then we should use it that way because after compilation, that'll be the only source of truth for the theme tokens that'll be accessible.
As for the example that you pointed out, raw value is still accessible during build time, so what you showed will be built-up as part of the generated css file and will not end-up in the runtime js bundle. So it'll work without issues.